jdaviz/tests/test_app.py

```diff
@@ -17,3 +17,58 @@ def test_get_tray_item_from_name():
 
     with pytest.raises(KeyError, match='not found in app'):
         app.get_tray_item_from_name('imviz-compass')
+
+
+def test_duplicate_data_labels(specviz_helper, spectrum1d):
+    specviz_helper.load_spectrum(spectrum1d, data_label="test")
+    specviz_helper.load_spectrum(spectrum1d, data_label="test")
+    dc = specviz_helper.app.data_collection
+    assert dc[0].label == "test"
+    assert dc[1].label == "test (1)"
+    specviz_helper.load_spectrum(spectrum1d, data_label="test_1")
+    specviz_helper.load_spectrum(spectrum1d, data_label="test")
+    assert dc[2].label == "test_1"
+    assert dc[3].label == "test (2)"

```

Comment:
For this test case, isn't no change more correct?
```suggestion
    assert dc[3].label == "test"
```

Reply:
Why do you say that? There are already two instances of test in data collection.

Reply:
Ah, nevermind. I missed the first 2 loading. Sorry for the noise!